tests/sc_dbpad_check.test/Makefile

```diff
@@ -0,0 +1,8 @@
+ifeq ($(TESTSROOTDIR),)
```

Comment:
Can this be part of the `ddl_no_csc2` test?

Reply:
Should I convert the tests to sqlite syntax if I move them here? If so, is it possible to add dbpad in an alter table statement this way?

Reply:
Or would I have to specify dbpad in the create table statement?

Reply:
Yes I think using the csc2 folder would be better in this case since the problem that led to this was a crash that specified dbstore = 0 when updating the byte array size instead of dbpad on alter table, and trying to test this case now with sqlite syntax on alter table (set default 0) for the byte array leads to a syntax error

Reply:
Updated with relocated tests